Modules/Core/Repositories/Cache/BaseCacheDecorator.php

```diff
@@ -187,7 +188,12 @@ protected function remember(\Closure $callback, $key = null)
             $store = $store->tags([$this->entityName, 'global']);
         }
 
-        return $store->remember($cacheKey, $this->cacheTime, $callback);
```

Comment:
Hm since there's a class property, how does it affect other methods?

Reply:
Ah I see ok, then we could remove the class property?

Reply:
We could, if we remove it we can just initialize it the method, just a sec and send the new PR

Reply:
sent